code-challenges/401challenges/src/main/java/challenges/tree/BinaryTree.java

```diff
@@ -66,4 +67,27 @@ public BinaryTree(T value){
         return (T[])list.toArray();
     }
 
+    public void breadthFirstTraversal(){
+       breadthTraversal(this.root);
```

Comment:
It might have been a good idea to check if `root == null` before calling the helper method